logic/verification_service.py

```diff
@@ -73,8 +95,148 @@ def verify_phone(self, req):
             verified=True)
         db.session.add(db_identity)
         db.session.commit()
+        # TODO: determine what the text should be
+        data = 'phone verified'
+        # TODO: determine claim type integer code for phone verification
+        claim_type = 10
+        signature = attestations.generate_signature(
+            web3, signing_key, req.eth_address, claim_type, data)
         return verification.VerifyPhoneResponse(
-            attestation=generate_signed_attestation(addr))
+            signature=signature, claim_type=claim_type, data=data)
+
+    def generate_email_verification_code(self, req):
+        addr = numeric_eth(req.eth_address)
+        db_code = VC.query \
+            .filter(VC.eth_address == addr) \
+            .filter(VC.email == req.email) \
```

Comment:
@tyleryasaka Should we query email case insensitively here and other places with similar query definition?

Reply:
Yeah, good call. Technically email addresses can be case sensitive before the `@`, but in practice they are always case insensitive. So it's a safe assumption. https://stackoverflow.com/a/9808332

Done 👍